src/auth/services/verify-login.service.ts

```diff
@@ -1,15 +1,14 @@
 import { Injectable } from '@nestjs/common';
-import { recover, hash } from 'eth-crypto';
 import { LoginRequest } from '../models/login-request.model';
+import { utils } from 'ethers';
 
 @Injectable()
 export class VerifyLoginService {
     public static readonly TIMESTAMP_TTL = 1000 * 60 * 15; // 15min
 
     /* Reconstructs plaintext message for the login request */
     public constructPlainSignature(timestamp: Date): string {
-        const payload = `oceanpearl.io - login @ ${timestamp.toISOString()}`;
-        return `\x19Ethereum Signed Message:\n${payload.length}${payload}`;
+        return `oceanpearl.io - login @ ${timestamp.toISOString()}`;
```

Comment:
question(blocking): You are returning a completly different string here. Was this intentinal or did you delete the prefix by mistake?

Reply:
Good question! Yes that is correct, the new framework automatically appends the prefix. 
So we no longer need to insert this manually.